src/components/IEEEFunctionInput.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="function-input">
+    <b>𝔽(</b>
+    <small>
+        base:<input type="number" v-model="base" size="2" v-bind="$attrs" />,
+        mantissaSize:<input type="number" v-model="mantissaSize" size="5" v-bind="$attrs" />,
+        eMin:<input type="number" v-model="eMin" size="5" v-bind="$attrs" />,
+        eMax:<input type="number" v-model="eMax" size="5" v-bind="$attrs" />,
+        denorm:<input type="text" v-model="denorm" size="5" v-bind="$attrs" />
+    </small>
+    <b>)</b>
+  </div>
```

Comment:
Future idea: Show what the format looks like:
- Maybe like |1 sign bit|5 exponent bits|1 implicit bit|10 mantissa bits|
- [Or like this](https://media.geeksforgeeks.org/wp-content/uploads/Single-Precision-IEEE-754-Floating-Point-Standard.jpg)

Reply:
Done